sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/aio/_path_client_async.py

```diff
@@ -736,7 +809,7 @@ async def _get_path_properties(self, **kwargs):
         :returns:
             Information including user-defined metadata, standard HTTP properties,
             and system properties for the file or directory.
-        :rtype: DirectoryProperties or FileProperties
+        :rtype: BlobProperties
```

Comment:
As mentioned in sync, the previous doc was correct.

Reply:
Also taking this feedback :)